### ISSUE TITLE
Collections Care Notice images need alt text

### ISSUE THREAD

**Issue by birbjam** [biocodellc/localcontexts_db#946]
On the Notices page, the images of the Collections Care Notices need descriptive alt text added to them for accessibility.

**Comment by corrieroe** [comment]
happy to help with this, if these don't exist yet :)

**Comment by birbjam** [comment]
Yes please @corrieroe !!

**Comment by corrieroe** [comment]
**Overall Notice**
Alt text:
Icon for Overall Collections Care Notice. Within a white square, black L.C. in middle of a circle made of dots.

**Authorization**
Alt text:
Icon for Authorization Notice. Abstract design within a white square. Black circle above a set of three curved black lines with one curved black line below.

**Belonging**
Alt text:
Icon for Belonging Notice. Abstract design within a white square. Three layered sets of black circles with two curved black lines representing three people.

**Caring**
Alt text:
Icon for Caring Notice. Abstract design within a white square. Two curved black lines surround a black circle.

**Gender Aware**
Alt text:
Icon for Gender Aware Notice. Abstract design within a white square. Three circles in a diagonal line: A black outlined circle, a black filled circle, and a striped black and white circle.

**Leave Undisturbed**
Alt text:
Icon for Leave Undisturbed Notice. Abstract design within a white square. A black circle in the center of two curved black lines which form a struck-through circle.

**Safety**
Alt text:
Icon for Safety Notice. Abstract design within a white square. A black circle in the center of three curved black lines which form a circle.

**Viewing**
Alt text:
Icon for Viewing Notice. Abstract design within a white square. A black circle in the center of two curved lines, forming an abstract eye.

**Withholding**
Alt text:
Icon for Withholding Notice. Abstract design within a white square. A black circle in the center of two curved lines.
